decidim-system/app/packs/src/decidim/system/api_users.js

```diff
@@ -0,0 +1,29 @@
+document.addEventListener("DOMContentLoaded", () => {
+  const copyToClipboard = document.getElementById("api-user-token")
+  if (!copyToClipboard) {
+    return
+  }
+
+  const tokenInput = document.getElementById("token");
+  if (!tokenInput) {
+    return
+  }
+  const copyToken = () => {
+    tokenInput.select();
+    navigator.clipboard.writeText(tokenInput.value);
+  }
+
+  const replaceText = () => {
+    let spanElement = document.createElement("span");
+    let parentElement = copyToClipboard.parentNode;
+    spanElement.textContent = "Copied!";
```

Comment:
The text here should be translated.

I also believe there is already a copy to clipboard component implemented in the core. Couldn't we reutilize existing code here? Especially now when the redesigned components are equally available to front-end and admin panel(s).

![Share modal in Decidim](https://github.com/user-attachments/assets/f30ac2d5-ed1d-4bf0-93bb-a3f60e4134da)